client/model.go

```diff
@@ -229,11 +229,25 @@ type Template struct {
 }
 
 type Environment struct {
-	Id            string `json:"id"`
-	Name          string `json:"name"`
-	ProjectId     string `json:"projectId"`
-	TemplateId    string `json:"templateId"`
-	WorkspaceName string `json:"workspaceName"`
+	Id                          string        `json:"id"`
+	Name                        string        `json:"name"`
+	ProjectId                   string        `json:"projectId"`
+	WorkspaceName               string        `json:"workspaceName,omitempty"`
+	RequiresApproval            bool          `json:"requiresApproval,omitempty"`
+	ContinuousDeployment        bool          `json:"continuousDeployment,omitempty"`
+	PullRequestPlanDeployments  bool          `json:"pullRequestPlanDeployments,omitempty"`
+	AutoDeployOnPathChangesOnly bool          `json:"autoDeployOnPathChangesOnly,omitempty"`
+	AutoDeployByCustomGlob      string        `json:"autoDeployByCustomGlob,omitempty"`
+	Status                      string        `json:"status"`
+	LatestDeploymentLogId       string        `json:"latestDeploymentLogId"`
+	LatestDeploymentLog         DeploymentLog `json:"latestDeploymentLog"`
```

Comment:
this model is what about BE sending back?

Reply:
yes, Environment is the environment response (its the same for create and update)
when I had special responses I made a struct just for them (for example deployResponse)